### DISCUSSION TITLE
Adding a Username Password Option for MQTT

### DISCUSSION THREAD

**Discussion by dervolki** [AK-Homberger/D1Mini-GasCounter#4] [Ideas]
Hello, 
first of all, thanks for this wunderfull idea. 
I would like kindly ask if there might be a chance to implement a option to add a username/password for the MQTT section.
I was trying to deal with it my self, but i am breaking more than making anything usefull. 
I am just not a SoftwareEngineer by any means. 

Thanks 
Volki

**Comment by AK-Homberger** [comment]
It would be possible to add username/password to MQTT. But why? Its in the local network and only non-sensitive data is trasmitted. I don't see any security risk here.

**Reply by dervolki** [reply]
Hello, 
Thanks for the very fast reply.
I am using a historicly grown home assistant setup with by default is using a username/password setup. 
Even though all the IOT stuff is in a seperate VLAN, I have to admit I would consider it not a bad idea to use credentials. 
i might be wrong. I might reconsider
